wiki/markdown/subscriptions.md

```diff
@@ -61,8 +61,7 @@ Below is a document from the database which contains a unique ID and an
 aggregated object.
```

Comment:
This sentence still reads as if the old structure was in the database. Maybe something like: "Below is a document containing an aggregated object. In addition to the extracted content from Eiffel events, Eiffel Intelligence has added an extra _id key with the value of the start event. This _id serves as an index in mongo DB and speeds up searches." would be better?

Reply:
done